cmd/frontend/graphqlbackend/search_results.go

```diff
@@ -881,6 +881,14 @@ func (r *searchResolver) Results(ctx context.Context) (*SearchResultsResolver, e
 	case *query.OrdinaryQuery:
 		return r.evaluateLeaf(ctx)
 	case *query.AndOrQuery:
+		// get settings to check if CaseSensitiveSearch is active or not and, if so, run transformer
+		settings, err := decodedViewerFinalSettings(context.Background())
```

Comment:
```suggestion
		settings, err := decodedViewerFinalSettings(ctx)
```
Actually needs to use `ctx`